### ISSUE TITLE
TypeError: can't set attributes of built-in/extension type 'object' when loading slots

### ISSUE THREAD

**Issue by diefans** [yaml/pyyaml#139]
```bash
python -c "$(cat << EOF
import yaml


class Foo:
    __slots__ = ('bar',)


foo = Foo()
foo.bar = 123
yaml.load(yaml.dump(foo))
EOF
)"
```

**Comment by hugovk** [comment]
@diefans Did https://github.com/diefans/pyyaml/commit/a74628731897a1612cb8890dff3c3a89bc0981a6 fix it?

**Comment by diefans** [comment]
@hugovk 
In version 3.12 `dump` and `load` use the unsafe stuff... So for the actual master branch you have to use:
```bash
python -c "$(cat << EOF
import yaml


class Foo:
    __slots__ = ('bar',)


foo = Foo()
foo.bar = 123
print(yaml.danger_load(yaml.danger_dump(foo)))
EOF
)"
```

This works for the example.

**Comment by diefans** [comment]
BTW I did not make a PR, since I was not able to write a unit test in an affordable amount of time - I am quite confused by the test setup...

**Comment by mezhaka** [comment]
@diefans Are you going to continues with that?  I stummbled upon this problem in my project and I wonder if it's going to be fixed.  At the same time I don't mind spending some time figuring out how to write a unit test.

**Comment by diefans** [comment]
@mezhaka my fix works for me so far (without unit tests), but at the moment I have no time left to spend on this.

**Comment by perlpunk** [comment]
Fixed in 5.3: https://pypi.org/project/PyYAML/5.3/
